data/iris.md

```diff
@@ -32,4 +35,6 @@ authorNPub: npub1g53mukxnjkcmr94fhryzkqutdz2ukq4ks0gvy5af25rgmwsl4ngq43drvk
 > 
 > • Create a personalized profile that showcases who you are and what you stand for
 > • Add a profile picture, bio, and additional information to your profile
-> • Show your followers what you're all about and make a lasting impression
\ No newline at end of file
+> • Show your followers what you're all about and make a lasting impression
+
+Also provides free NIP-05 verification service
```

Comment:
NIP-05 is not about verification even though some of the services try to sell it as such. It's only **id**entification. Especially free providers don't even "verify" they get paid. They just want to get popular to later charge.

```suggestion
Also provides free NIP-05 addresses.
```

Reply:
fixing.